src/sidebar/components/group-list-item.js

```diff
@@ -89,9 +86,7 @@ function GroupListItem({
       onToggleSubmenu={toggleSubmenu}
       submenu={
         <Fragment>
-          {/* FIXME-A11Y */}
-          {/* eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/click-events-have-key-events */}
-          <ul onClick={collapseSubmenu}>
+          <ul>
```

Comment:
@LMS007 - you're right that there isn't any observable behavior change as a result of removing the `onClick` handler, but the reason is not that the `Menu` component handles this, but that the top level `GroupList` component does:

https://github.com/hypothesis/client/blob/7de51cfa2c7ea1bc56d4fb1d24fc9d64555bbec3/src/sidebar/components/group-list.js#L93-L99

If you remove the `onOpenChanged` prop, you'll see that if you expand the menu for a group and then click one of the items in it, that group will still be expanded the next time that the menu is opened. From memory, I think the `onOpenChanged` handler was added after the `onClick={collapseMenu}` logic and I hadn't realized that it made the existing code obsolete.

So to summarize, the change here makes sense, but I wanted to note the correct explanation for posterity.

Reply:
gotcha! thanks Rob